demos/patch/hdiv_riesz_star.py.rst

```diff
@@ -9,7 +9,8 @@ Here, we demonstrate how to do this in the former case. ::
 
   from firedrake import *
 
-  base = UnitCubeMesh(2, 2, 2)
+  dparams = {"overlap_type": (DistributedMeshOverlapType.VERTEX, 1)}
```

Comment:
Please can you add the text explaining what this line does from the poisson_mg_patches demo?